media-gfx/fontforge/fontforge-20220308.recipe

```diff
@@ -0,0 +1,139 @@
+SUMMARY="An outline font editor"
+DESCRIPTION="An outline font editor that lets you create your own OpenType, \
+TrueType, CID-keyed, multi-master fonts and more... or simply edit existing \
+ones."
+HOMEPAGE="https://fontforge.github.io/"
+COPYRIGHT="2000-2018 George Williams
+	The FontForge Project"
+LICENSE="GNU GPL v3
+	BSD (3-clause)"
+REVISION="4"
+SOURCE_URI="https://github.com/fontforge/fontforge/releases/download/$portVersion/fontforge-$portVersion.tar.xz"
+CHECKSUM_SHA256="01e4017f7a0ccecf436c74b8e1f6b374fc04a5283c1d68967996782e15618e59"
+ADDITIONAL_FILES="fontforge.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2 !x86"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	fontforge$secondaryArchSuffix = $portVersion
+	app:fontforge
```

Comment:
`cmd:fontforge` is not provided. It should probably be added back. There are use-cases for fontforge as a command line tool.